### PR TITLE
docs(connector-twilio-sms): add Twilio SMS README.md (en-US)

### PR DESCRIPTION
**Pull request by darcyYe**
## Summary

Add Twilio SMS README.md (en-US).

## Testing

N/A

### CODE REVIEW

File: packages/connector-twilio-sms/README.md [hunk 1]
```diff
@@ -1,2 +1,73 @@
...
+## Set up sender's phone numbers
+
+Go to the [Twilio console page](https://console.twilio.com/) and sign in with your Twilio account.
+
+Switch to _Develop_ tab on the top of the sidebar, and then unfold _Phone Numbers_ and _Messaging_.
+
+Click _Buy a number_ under _Phone Numbers_ -> _Manage_, and you can buy phone number(s) as your sender's phone number.
+
+Go to _Messaging_ -> _Services_ and click _Create Messaging Service_ button on the top-right corner of the panel.
+    1. Name your messaging service and type in the name in _Messaging Service friendly name_ field. Choose the proper purpose, in our case, is `Notify my users`.
```
Comment (simeng-li) [inline packages/connector-twilio-sms/README.md:18]:
```suggestion
    1. Give a friendly service name and choose *Verify Users* as our service purpose 
```

File: packages/connector-twilio-sms/README.md [hunk 2]
```diff
@@ -1,2 +1,73 @@
...
+    2. Go to _Add Senders_, choose `Phone Number` as _Sender Type_, and continue. Select the sender you want to link to the messaging service that is currently being created and move on by clicking _Add Phone Numbers_. Need to notice that a phone number (or sender) can only be bound to one messaging service.
```
Comment (simeng-li) [inline packages/connector-twilio-sms/README.md:19]:
```suggestion
    2. Following the next step, choose `Phone Number` as _Sender Type_, and link the phone number we just claimed to this service. (Note: Each phone number can only be linked with one messaging service)
```

Comment (simeng-li) [reply packages/connector-twilio-sms/README.md:19]: image
 
 skipped?

Comment (simeng-li) [reply packages/connector-twilio-sms/README.md:19]: Also need the summary sentence, sth like, `goes to the next step once we have your SMS service set up`

File: packages/connector-twilio-sms/README.md [hunk 3]
```diff
@@ -1,2 +1,73 @@
...
+
+## Get account credentials
+
```
Comment (simeng-li) [inline packages/connector-twilio-sms/README.md:22]: Need a brief introduction, we will need the API credentials to make the connector work. Then:

Click on the account menu in the top-right corner, then go to the _API keys & tokens_ page to consume your `Account SID` and `Auth token`. 

Back to the _Messaging_ -> **_Services_** settings page, and find the `Sid` of your service.

File: packages/connector-twilio-sms/README.md [hunk 4]
```diff
@@ -1,2 +1,73 @@
...
+## Set up sender's phone numbers
+
+Go to the [Twilio console page](https://console.twilio.com/) and sign in with your Twilio account.
+
+Switch to _Develop_ tab on the top of the sidebar, and then unfold _Phone Numbers_ and _Messaging_.
+
+Click _Buy a number_ under _Phone Numbers_ -> _Manage_, and you can buy phone number(s) as your sender's phone number.
+
+Go to _Messaging_ -> _Services_ and click _Create Messaging Service_ button on the top-right corner of the panel.
```
[the remark this answers is not in the document]
Comment (IceHe) [reply packages/connector-twilio-sms/README.md:17]: @simeng-li `*_Services_*` → *_Services_*

what you expecte is `**_Services_**` → **_Services_**?

File: packages/connector-twilio-sms/README.md [hunk 5]
```diff
@@ -1,2 +1,73 @@
-### Twilio SMS README
-placeholder
+# Twilio short message service connector
+
+Twilio provides programmable communication tools for making and receiving phone calls, sending and receiving text messages, and performing other communication functions using its web service APIs. We can take advantage of its sending text message function to send a "verification code" (or in other words, passcode).
```
[the remark this answers is not in the document]
Comment (IceHe) [reply packages/connector-twilio-sms/README.md:3]: maybe better explain the `passcode` and `verification code` terms are the same.

**Review by simeng-li — APPROVED** [review]
overall LGTM

File: packages/connector-twilio-sms/README.md [hunk 6]
```diff
@@ -1,2 +1,79 @@
...
+
+## Register Twilio account
+
+Create a new account at [Twilio](https://www.twilio.com). (Jump to the next step if you already have one.)
+
+## Set up senders' phone numbers
+
+Go to the [Twilio console page](https://console.twilio.com/) and sign in with your Twilio account.
+
+Click on the _Develop_ tab on the top of the sidebar, and then unfold _Phone Numbers_ and _Messaging_.
```
[the remark this answers is not in the document]
Comment (darcyYe) [reply packages/connector-twilio-sms/README.md]: Fixed, will follow this suggestion.

**Pull request merged**